contracts/bridge/BridgeTransferERC721.sol

```diff
@@ -84,6 +84,11 @@ contract BridgeTransferERC721 is BridgeTokens, IERC721BridgeReceiver, BridgeTran
     {
         require(isRunning, "stopped bridge");
 
+        (bool success, bytes memory uri) = _tokenAddress.call(abi.encodePacked(ERC721Metadata(_tokenAddress).tokenURI.selector, abi.encode(_tokenId)));
+        if (success == false) {
+            uri = "";
+        }
+
```

Comment:
This code is newly added unlike original code https://github.com/klaytn/klaytn/pull/249/files?diff=split&w=0.
Also this false exception case can be tested in `TestBridgeManagerERC721_notSupportURI`.

It's good change. Thanks.